x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/common/card_content_asset_panel.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo, type PropsWithChildren } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { OnboardingCardContentPanel } from './card_content_panel';
+import { useCardContentAssetPanelStyles } from './card_content_asset_panel.styles';
+import { CardSelectorListItemAssetType } from './card_selector_list';
+
+export const OnboardingCardContentAssetPanel = React.memo<
+  PropsWithChildren<{
+    asset: { type: CardSelectorListItemAssetType; source: string; alt: string };
```

Comment:
Can we create and export a type for the `asset`? Also, this `type` prop has nothing to do with _CardSelectorList_, it should be `type: OnboardingCardContentAssetType`, and defined here as well